services/connected_account.yaml

```diff
@@ -9,35 +9,20 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: Account Name Metadata for your Account
+          default: Required
```

Comment:
Was this change intentional?  I'm kinda confused by it

Reply:
Yes. We wanted to make it clearer to the user which CFN parameters are required. This section is for the Labels that appear in the CFN "Create Stack" Console